srcpkgs/folks/template

```diff
@@ -15,15 +16,9 @@ license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/Folks"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
 checksum=3df1c663b82fe211aead2b31cae1fc41d124e85436375762f6745c45ba46aa5d
-nocross="Needs Vala, which needs evolution-dataserver GIR"
 
 # NOTE: Vala build opt is broken because upstream doesn't package all necessary
 # files in the tarballs, .gir files are missing.
-build_options="gir vala"
```

Comment:
you've removed build options but wasn't revbumped.

Reply:
Woops :)